common/djangoapps/entitlements/filters.py

```diff
@@ -0,0 +1,41 @@
+from django_filters import rest_framework as filters
+
+from .models import CourseEntitlement
+
+
+class CharListFilter(filters.CharFilter):
+    """ Filters a field via a comma-delimited list of values. """
+
+    def filter(self, qs, value):  # pylint: disable=method-hidden
+        if value not in (None, ''):
+            value = value.split(',')
+
+        return super(CharListFilter, self).filter(qs, value)
+
+
+class UUIDListFilter(CharListFilter):
```

Comment:
Is this the same as the one in edx/course-discovery? Consider moving it to https://github.com/edx/edx-drf-extensions so that we can share the code.

Reply:
Yes this was a copy and paste. Can we discuss this when you get in?

Reply:
Per our discussion I am going to create a story for this work and add it to the Helio backlog